src/tests/Dictionary/DictionaryContainer.test.jsx

```diff
@@ -88,7 +88,7 @@ describe('DictionaryOverview', () => {
   });
 
   it('should handle HEAD version release', () => {
-    const props = {
+    const  props = {
```

Comment:
@dessHub , why the extra space before `props`?

Reply:
Fixed it.